user/user.js

```diff
@@ -0,0 +1,43 @@
+const constants = require('../utils/constants')
+
+const getUser = (req, res) => {
+  const token = req.body.token;
+  const userId = createUserId(token);
```

Comment:
No need to have both createUserId and createUserData. Combine please into getUserData method with token check for 
 now and  get user data from DB in future.

Reply:
fixed